core/computed/metrics/lcp-load-start.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {makeComputedArtifact} from '../computed-artifact.js';
+import {NavigationMetric} from './navigation-metric.js';
+import {LighthouseError} from '../../lib/lh-error.js';
+import PrioritizeLcpImage from '../../audits/prioritize-lcp-image.js';
+import {NetworkRecords} from '../network-records.js';
+import {LargestContentfulPaint} from './largest-contentful-paint.js';
+import {ProcessedNavigation} from '../processed-navigation.js';
+
+class LCPLoadStart extends NavigationMetric {
+  /**
+   * @param {LH.Artifacts.NetworkRequest} record
+   * @return {number}
+   */
+  static getTimestamp(record) {
+    return record.networkRequestTime * 1000;
+  }
+
+  /**
+   * @param {LH.Artifacts.NavigationMetricComputationData} data
+   * @param {LH.Artifacts.ComputedContext} context
+   * @return {Promise<LH.Artifacts.Metric>}
+   */
+  static async computeSimulatedMetric(data, context) {
+    const processedNavigation = await ProcessedNavigation.request(data.trace, context);
+    if (processedNavigation.timings.largestContentfulPaint === undefined) {
+      throw new LighthouseError(LighthouseError.errors.NO_LCP);
+    }
+
+    const lcp = await LargestContentfulPaint.request(data, context);
+    const observedMetric = await this.computeObservedMetric(data, context);
+
+    const throttleRatio = lcp.timing / processedNavigation.timings.largestContentfulPaint;
+    const timing = observedMetric.timing * throttleRatio;
+    return {timing};
+  }
```

Comment:
We are still computing a simulated metric, but it isn't really a "lantern" metric.